packages/mail-client/src/mail-client.ts

```diff
@@ -194,19 +194,27 @@ async function sendMailInSequential<T extends MailConfig>(
   mailConfigs: T[]
 ): Promise<MailResponse[]> {
   const response: MailResponse[] = [];
-  for (const mailConfigIndex in mailConfigs) {
-    logger.debug(
-      `Sending email ${mailConfigIndex + 1}/${mailConfigs.length}...`
-    );
-    response[mailConfigIndex] = await transport.sendMail({
-      ...mailConfigsFromDestination,
-      ...mailConfigs[mailConfigIndex]
-    });
-    logger.debug(
-      `...email ${mailConfigIndex + 1}/${mailConfigs.length} for subject "${
-        mailConfigs[mailConfigIndex].subject
-      }" was sent successfully.`
-    );
+  for (const [mailConfigIndex, mailConfig] of mailConfigs.entries()) {
```

Comment:
[pp] Maybe use map also here to keep them consistent?

Reply:
But it will no longer be sequential then

Reply:
I think, it will still be sequential even if you use a `map` as you don't have an async keyword used in this function block.